gravitee-apim-rest-api/gravitee-apim-rest-api-management-v4/gravitee-apim-rest-api-management-v4-rest/src/main/java/io/gravitee/rest/api/management/v4/rest/resource/connector/EntrypointsResource.java

```diff
@@ -46,20 +50,23 @@ public class EntrypointsResource {
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @Permissions({ @Permission(value = RolePermission.ORGANIZATION_ENTRYPOINT, acls = RolePermissionAction.READ) })
```

Comment:
ORGANIZATION_ENTRYPOINT is not meant to be used with connectors, it's about the entrypoints of the gateway when using sharding tags.
I would recommend creating a dedicated permission for connectors, or maybe do not use permissions at all